packages/fxa-settings/src/components/Settings/PageSettings/index.tsx

```diff
@@ -104,6 +148,11 @@ export const PageSettings = (_: RouteComponentProps) => {
         />
       </div>
       <div className="flex flex-col flex-7 max-w-full gap-8 mt-10">
+        {isInactiveVerifiedBannerVisible && (
+          <div className="mx-auto">
+            <Banner {...bannerProps} />
+          </div>
+        )}
```

Comment:
Sending you a diff using `alertBar` instead! I think then you can revert the Banner changes too.